src/main/scala/GameRunner.scala

```diff
@@ -0,0 +1,113 @@
+import GameHelpers._
+import GamePrompts._
+import scala.util.Random
+
+object GameRunner extends App {
+
+  def gameStart(): Unit = {
+    println()
+    println("Welcome to \u001b[31;1mScala Scrabble Practice")
+    println()
+    val name: String = promptName()
+    println()
+    println("Hello " + name + ". \u001b[0mThe Human plays against the Computer.")
+    println()
+    val level: String = promptLevel()
+
+    val letterBag = Bag.initialise()
+    val human = Player(name, Nil, Nil, false)
+    val computer = Player("The Computer", Nil, Nil, false)
+    val newGame = Game(letterBag, human, computer, level)
+
+    humanTurn(newGame)
+  }
+
+  def computerTurn(game: Game): Unit = {
+    checkGameOver(game)
+
+    val computer = game.computer
+    val (rack, bag) = fillRack(computer.rack, game.bag)
+    displayRack(computer.name, rack)
+
+    val word = playWord(rack, game.level)
+
+    word match {
+      case Some(w) =>
+        println(computer.name + s" has played: '$w'")
```

Comment:
If you're using string interpolation here, then you can avoid doing "" + "" and put it all in one string 👍